include/Graphs/CFLGraph.h

```diff
@@ -45,12 +46,19 @@ typedef GenericEdge<CFLNode> GenericCFLEdgeTy;
 
 class CFLEdge: public GenericCFLEdgeTy
 {
+    static u32_t offset;
```

Comment:
Do we need this `offset`? This is a general CFLEdge and should not have information from a particular graph like PAG.

Reply:
Yes, that should be deleted.